Src/EmbeddedScripts.CSharp.Shared/ResolvingCodeGenerator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Linq;
+using EmbeddedScripts.Shared;
+
+namespace EmbeddedScripts.CSharp.Shared
+{
+    public static class ResolvingCodeGenerator
```

Comment:
Do I really need this class or it would be better to move these methods to `CodeBuilder.BuildCode()`?

Reply:
CodeBuilder should be responsible for everything concerning C# code generation then